arrow-typeclasses/src/main/kotlin/arrow/typeclasses/Show.kt

```diff
@@ -0,0 +1,55 @@
+package arrow.typeclasses
+
+import arrow.*
+
+
+/**
+ * A type class used to get a textual representation for an instance of type [F] in a type safe way.
+ *
+ */
+@typeclass
+interface Show<in F> : TC {
```

Comment:
The convention in type params here is to use `A` instead of `F` since this type param referes to a monomorphic applied type and not to an unapplied higher kinded type constructor